contrib/ray/raycluster_example.yaml

```diff
@@ -2,48 +2,29 @@ apiVersion: ray.io/v1
 kind: RayCluster
 metadata:
   name: kubeflow-raycluster
+  namespace: kubeflow
```

Comment:
This is not the right namespace. It should be the customer/user namespace

Reply:
Can you explain bit more